CHANGELOG.md

```diff
@@ -60,6 +61,7 @@
   - increased smart contract module size limit, 512kB
   - a number of cryptographic primitives
 - Node can now be stopped during out of band catchup by using signals, SIGINT and SIGTERM.
+- Upgrade resolver to version `lts-19.6` and ghc to version `9.0.2`.
```

Comment:
The changelog is not for these kinds of changes. This is not visible to any users.

Reply:
done